charts/rancher-istio/1.5.902/Chart.yaml

```diff
@@ -14,4 +14,4 @@ name: rancher-istio
 sources:
 - http://github.com/istio/istio
 tillerVersion: '>=2.7.2-0'
-version: 1.5.901
+version: 1.5.902
```

Comment:
Because this is 2.6, I think you have to make this 1.5.920. That way several versions in 2.5 can still use 1.5.902-1.5.919

Reply:
Updated to be 1.5.920 so that our 2.5 branch can continue to have additional fixes - I went ahead and left it as a separate commit, but will merge the commits once review is completed and PR is approved